client/components/Accessories.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+class Accessories extends React.Component {
```

Comment:
if not using state, this can probably be a function component (instead of a class component)